TCAT/Supporting Files/Constants.swift

```diff
@@ -139,6 +139,7 @@ struct Constants {
 
         static let busStopDetailCellIdentifier = "BusStopCell"
         static let largeDetailCellIdentifier = "LargeCell"
+        static let notificationToggleCellIdentifier = "NotifToggleCell"
```

Comment:
nit, but prefer `NotificationToggleCell`, no reason not to be verbose here

Reply:
If we change that cell name, should probably also change the other constants similarly to remain consistent.